Sources/tart/MACAddressResolver/Leases.swift

```diff
@@ -107,7 +107,7 @@ class Leases {
     return rawLeases
   }
 
-  func resolveMACAddress(macAddress: MACAddress) throws -> IPv4Address? {
+  func ResolveMACAddress(macAddress: MACAddress) throws -> IPv4Address? {
```

Comment:
Why change the capitalization?

Reply:
So it's the same name as for the `arp`. I though about introducing a Protocol too but ended up not doing that. 🤷